src/music/Samples.ts

```diff
@@ -0,0 +1,58 @@
+// TODO: do this better
+// This is necessary for decoding sample data, but there has to be a better way to do this.
+const tempAudioContext = new AudioContext();
```

Comment:
Can you inject this from the main `AudioContext` somehow?

Reply:
It would require more restructuring than it's theoretically worth in my view, though I can be convinced otherwise.

Reply:
Perhaps if you left `Drumkit` as a class, and injected the `AudioContext` through the constructor instead of passing it via `scheduleHit`?

Reply:
This is fine, I can do this after merge